### PR TITLE
gnome-extra/gnome-browser-connector: enable py3.12

### PR DESCRIPTION
**Pull request by roarc0**
enable py3.12

### CODE REVIEW

**Comment by nE0sIghT** [comment]
Fixes: https://bugs.gentoo.org/929602

P.S.: I didn't tested it yet

**Comment by roarc0** [comment]
I've compiled it with the 3.12 flag then rebooted chrome and extensions.gnome.org shows my currently installed extensions. When I try to install one it just downloads the extension archive.

File: gnome-extra/gnome-browser-connector/gnome-browser-connector-42.1.ebuild [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{9..12} )
```
Comment (mgorny) [inline gnome-extra/gnome-browser-connector/gnome-browser-connector-42.1.ebuild:5]: Please remove 3.9 while at it.

**Review by mgorny — CHANGES_REQUESTED** [review]
Need signoff on this.

**Pull request closed without merging**